lib/service_locator.dart

```diff
@@ -1,13 +1,16 @@
-import 'package:get_it/get_it.dart';
-
 import 'data/repositories/repository.dart';
 import 'data/repositories/repository_impl.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
 import 'data/services/network_service.dart';
 
-final getIt = GetIt.instance;
+final _networkServiceProvider = Provider<NetworkService>(
```

Comment:
pls add todo to remove it in the future to not forget

Reply:
+